PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Schematic Level Requirements 
+- [ ] Is proper noise resistance given to all peripheral devices (bypass caps and coils/ferrites)?  
+- [ ] Is proper ESD protection given to all MCU input pins (zener diodes)?  
+- [ ] Is proper power protection given to peripheral devices (zener diodes)?  
+- [ ] Are peripheral units used properly (reading datasheet)?  
+- [ ] Are testing points added at useful places?  
+- [ ] Is there proper short to GND protection at MCU outputs (inline resistors)?  
+- [ ] Do ADC inputs have caps?  
+- [ ] Are ADC inputs biased so there is room above expected value to determine if value is being overflowed?  
+- [ ] Are LED's located at useful places (comm, power, debugging, extra GPIO)?  
+- [ ] Are parts chosen easy to collect?  
+- [ ] Are parts chosen easy to solder?  
+- [ ] Is there reverse polarity protection on inputs?  
+- [ ] Do parts in the schematic match the actual part datasheet?
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## BOM Requirements
+- [ ] **Bill Of Materials**: is there an Excel Sheet BOM and a Mouser Cart link?
+- [ ] Are *all* the correct parts in the BOM?
+- [ ] Is each part in the Mouser Cart in stock (or do we have the part in stock)?
+- [ ] Does each part in the Mouser Cart have a margin for part quantity?
+- [ ] Does every component in the schematic have a footprint, mouser link/datasheet, and reference number associated with it?
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Layout Level Requirements 
+### 2D Spacing
+- [ ] The components are spaced out at an optimal distance.
+    - All components can be easily hand-soldered.
+    - IPC-SM-782A Standard requires a minimum distance of 1.0mm from edge to edge.
+- [ ] Components that are in parallel with each other are spaced out at an equal distance when possible.
+- [ ] The components are aligned with each other when possible.
+- [ ] Components are grouped based off of functionality.
+    - E.g. all components for CAN should be grouped.
+- [ ] Bypass capacitors are less than 1cm away from their respective IC's power pins.
+
+### 3D Spacing
+- [ ] Layout of components have been placed with mounting location and usage of the board in mind.
+    - Are PCBs going to be stacked on above/below this board? Are tall components placed accordingly?
+    - Are buttons and lights easily accessible and viewable?
+    - When mounted into the car, are the heights of the components and connectors accounted for?
+- [ ] I am not reading every word and I need to start this checklist over.
+    - Please uncheck everything you have checked and start over.
+- [ ] Location of connectors and wires going out of the board will prevent messy wiring.
+    - Are all the wires that are going in the same direction placed on the same edge of the board?
```

Comment:
Not sure where to put this, but something to check for is if power connectors are standardized
![image](https://user-images.githubusercontent.com/31521956/236360939-c07821d6-40ea-4c11-8fda-ef8360ba0bec.png)

e.g. left of the molex is +12V and right is GND - RELATIVE to the CONNECTOR
I've goofed up some boards where I flip the .sch symbol and it turns out the connector I make for it in reality is reversesd.

Reply:
![image](https://user-images.githubusercontent.com/55002301/236367569-21837920-6ad3-4339-b168-262393b7f004.png)
We have symbol for power connector.